test/libsolidity/smtCheckerTests/external_calls/call_with_value_3.sol

```diff
@@ -0,0 +1,15 @@
+contract C {
+	function g(address payable i) public {
+		require(address(this).balance > 100);
+		i.call{value: 20}("");
+		assert(address(this).balance > 0); // should hold
+		// Disabled due to Spacer nondeterminism
+		//assert(address(this).balance == 0); // should fail
+	}
+}
+// ====
+// SMTEngine: all
+// ----
+// Warning 9302: (95-116): Return value of low-level calls not used.
+// Warning 6328: (120-153): CHC: Assertion violation might happen here.
+// Warning 4661: (120-153): BMC: Assertion violation happens here.
```

Comment:
Again, 120 is the line with "should hold".

Reply:
same, chc `might happen` means it couldn't solve it, bmc `happens` is a false positive

Reply:
So, how can we know the feature even works? :P I think all tests have false positives or non-determinism.

Reply:
Not all though. Combining all the new tests there are a few correct `holds` and a few correct `fails`. But before disabling all the ones that I did, all the tests actually passed without disabling any of them, except for osx.

Reply:
ok